### ISSUE TITLE
Initial value not displayed on first build

### ISSUE THREAD

**Issue by daniel-v** [TinoGuo/pin_input_text_field#15]
Sample:

```dart
final _controller = TextEditingController(text:'654');
PinInputTextFormField(
  pinLength: 6,
  controller: _controller,
);
```

When PinInputTextFormField is built the first time, `654` won't show up. Upon focus, it does. This occurs both with `controller` and `initialValue` properties. I believe the cause of the probalem is that `_text` is set to `null` when the first build runs.

**Comment by TinoGuo** [comment]
Thank you for your feedback. I merged your code and published the new version [2.0.2](https://pub.dartlang.org/packages/pin_input_text_field)

**Comment by daniel-v** [comment]
Thanks!
